### PR TITLE
#2q3wpje text select

### PR DESCRIPTION
**Pull request by IzzXr**
## Task

Issue: #2q3wpje

## Decision

> Please describe task description in more detail.
> If it was a bug, then describe what was causing the problem.

add style

## Screenshots

> If you changed the UI, then you need to attach a screenshot/video.

### CODE REVIEW

**Comment by alexeyshirko** [comment]
Task linked: [CU-2q3wpje Text select](https://app.clickup.com/t/2q3wpje)

File: src/views/menu/layout/components/Navigation/style/styles.module.scss [hunk 1]
```diff
@@ -18,6 +19,7 @@
       text-align: center;
       font-size: var(--cops-font-size-big);
       font-family: var(--cops-font-family-roboto);
+      font-weight: var(--cops-font-weight-bold);
```
Comment (alexeyshirko) [inline src/views/menu/layout/components/Navigation/style/styles.module.scss:22]: why do we need to change the thickness of the text here? These are changes from another branch

Comment (IzzXr) [reply src/views/menu/layout/components/Navigation/style/styles.module.scss:22]: I made a mistake when creating a branch, I realized my mistake but i couldn't fix it

Comment (alexeyshirko) [reply src/views/menu/layout/components/Navigation/style/styles.module.scss:22]: switch to this branch, remove this line and start a new commit

**Review by alexeyshirko — APPROVED** [review]

**Pull request merged**